forge-gui/res/cardsfolder/upcoming/nobles_purse.txt

```diff
@@ -0,0 +1,9 @@
+Name:Noble's Purse
+ManaCost:2
+Types:Artifact
+K:ETBReplacement:Other:EntersTapped
+SVar:EntersTapped:DB$ Tap | Defined$ Self | ETB$ True | SubAbility$ DBAddCounter | SpellDescription$ CARDNAME enters the battlefield tapped and with three coin counters on it.
+SVar:DBAddCounter:DB$ PutCounter | ETB$ True | Defined$ Self | CounterType$ COIN | CounterNum$ 3
+A:AB$ Token | Cost$ T SubCounter<1/COIN> | TokenAmount$ 1 | TokenScript$ c_a_treasure_sac | TokenOwner$ You | SpellDescription$ Create a Treasure token.
```

Comment:
add the reminder text to `SpellDescription`

`TokenAmount$ 1` and `TokenOwner$ You` are defaults